packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -149,14 +150,26 @@ def _serialize_flattened_body(body_parameter: BodyParameter) -> List[str]:
     return retval
 
 
-def _serialize_json_model_body(body_parameter: BodyParameter) -> List[str]:
+def _serialize_json_model_body(
+    body_parameter: BodyParameter, parameters: List[ParameterType]
+) -> List[str]:
     retval: List[str] = []
     if not body_parameter.property_to_parameter_name:
         raise ValueError(
             "This method can't be called if the operation doesn't need parameter flattening"
         )
 
     retval.append(f"if {body_parameter.client_name} is _Unset:")
+    for p in parameters:
```

Comment:
i'm going to streamline the logic just a little bit, but overall this looks v good. Thanks so much!

Reply:
actually i think this logic is better than the `any()` stuff we were talking about. COmmitting, thanks!